core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -4732,67 +4549,6 @@ class ReplicaManagerTest {
     }
   }
 
-  @Test
-  def testPartitionMetadataFileCreatedAfterPreviousRequestWithoutIds(): Unit = {
```

Comment:
@jolshan I believe this and`testPartitionFetchStateUpdatesWithTopicIdChanges` are no longer necessary since KRaft always requires topic ids to be set and 4.0 only supports upgrades from clusters already running in kraft mode. Do you agree or are there reasons to try to make these tests work?

Reply:
I think your conclusion makes sense.